content/graphing/infrastructure/livecontainers.md

```diff
@@ -17,17 +17,35 @@ further_reading:
 
 [Datadog Live Containers][3] enable real-time visibility into all containers across your environment.
 
-Taking inspiration from bedrock tools like *htop* and *ctop*, Live Containers give you complete coverage of your container infrastructure, in a continuously updated table with resource metrics at two-second resolution and faceted search.  
+Taking inspiration from bedrock tools like *htop* and *ctop*, Live Containers give you complete coverage of your container infrastructure, in a continuously updated table with resource metrics at two-second resolution and faceted search.
 Coupled with Datadog’s integrations with [Docker][4], [Kubernetes][5], [ECS][6], and other container technologies, plus our built-in tagging of dynamic components, this new Live Container view provides a detailed overview of your containers’ health, resource consumption, and deployment in real time:
 
-{{< img src="graphing/infrastructure/livecontainers/LiveContainersWithSummaries.png" alt="Live containers with summaries" responsive="true" popup="true">}} 
+{{< img src="graphing/infrastructure/livecontainers/LiveContainersWithSummaries.png" alt="Live containers with summaries" responsive="true" popup="true">}}
 
 ## Installation
 
 After deploying the [Docker Agent][7], no other configuration is necessary.
 
 Note that for collecting Container information in the standard install rather than with the [Docker Agent][7], the `dd-agent` user needs to have permissions to access **docker.sock**.
 
+### Include/Exclude containers
+
+It is possible to include and/or Exclude containers from real-time collection:
+
+- Exclude containers either via passing the environment variable `DD_AC_EXCLUDE` or adding `ac_exclude:` in your `datadog.yaml` main configuration file.
+- Include containers either via passing the environment variable `DD_AC_INCLUDE` or adding `ac_include:` in your `datadog.yaml` main configuration file.
+
+Both arguments take an **image name** as value; regexp are also supported.
+
+For example, to exclude all debian images except containers with a name starting with *frontend*, add these two configuration lines in your `datadog.yaml`file:
+
+```
+ac_exclude: ["image:debian"]
+ac_include: ["name:frontend.*"]
+```
+
+**Note:** For Agent v5, instead of including the above in the `datadog.conf` main configuration file, You have to explicitly add a `datadog.yaml` file to `/etc/datadog-agent/datadog.yaml` as the Process Agent relies on configuration options set here. This configuration only excludes containers from real-time collection **not** from Autodiscovery.
```

Comment:
Pushed a couple nit fixes, but I believe this section is incorrect. Can we verify?

Reply:
@jeremy-lq lines  31-46 are pulled straight from the agent's datadog.yaml. Line 47+ was confirmed by Burrito and tested on my end as the result of confusion as to set up for agent 5. it's all agent 5 specific.